RHIDP-4780: Document per-ConfigMap/Secret configuration of mountPath (#842)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/ref-release-notes-new-features.adoc

[id="enhancement-rhidp-2200"]
== Added an individual `mountPath`

> This update adds an individual `mountPath` for link:{configuring-book-url}#proc-mounting-additional-files-in-your-custom-configuration-using-rhdh-operator[extra configmaps or secrets].

Reviewer: ```suggestion
This update adds an individual `mountPath` for link:{configuring-book-url}#proc-mounting-additional-files-in-your-custom-configuration-using-rhdh-operator[extra ConfigMaps or Secrets].
```

> This update adds an additional individual `mountPath` for link:{configuring-book-url}#proc-mounting-additional-files-in-your-custom-configuration-using-rhdh-operator[extra configmaps or secrets].

Reviewer: ```suggestion
This update adds an individual `mountPath` for link:{configuring-book-url}#proc-mounting-additional-files-in-your-custom-configuration-using-rhdh-operator[extra configmaps or secrets].
```
to remove the redundancy
confirmed that the link works

[id="feature-rhidp-4414"]
== Identifying Backstage flavor for plugins by using the `developerHub.flavor` field

With this update, you can use the `developerHub.flavor` field to identify whether plugins are running on {product-very-short}, {rhtap-very-short}, or vanilla Backstage, as shown in the following example:

.`app-config.yaml` fragment with the `developerhub.flavor` field

[source,yaml,subs=&#34;quotes&#34;]
----
developerHub:

>   flavor: &lt;flavor&gt;;

Reviewer 2: ```suggestion
  flavor: <flavor>
```

----

`flavor`::
Identify the flavor of Backstage that is running. Default value: `rhdh`

[id="feature-rhidp-4419"]
== Ability to manage PVCs in RHDH Operator

> You can now mount directories from pre-created PersistentVolumeClaims (PVCs) using the `spec.application.extraFiles.pvcs` field, while configuring RHDH Operator.

Reviewer 2: ```suggestion
You can now mount directories from pre-created PersistentVolumeClaims (PVCs) using the `spec.application.extraFiles.pvcs` field, while configuring {product-very-short} Operator.
```
